mifospay/src/main/java/org/mifos/mobilewallet/mifospay/bank/ui/SetupUpiPinActivity.kt

```diff
@@ -78,41 +50,41 @@ class SetupUpiPinActivity : BaseActivity(), SetupUpiPinView {
         type = b.getString(Constants.TYPE)
         if (type == Constants.CHANGE) {
             mSetupUpiPinPresenter!!.requestOtp(bankAccountDetails)
-            mFlDebitCard!!.visibility = View.GONE
-            mCvDebitCard!!.visibility = View.GONE
+            binding.flDebitCard!!.visibility = View.GONE
+            binding.cvDebitCard!!.visibility = View.GONE
```

Comment:
why we are using the binding here? and why it is not converted into compose.

Reply:
now completely migrated whole screen to compose .